Scripts/gen_CosmicStage2.sh

```diff
@@ -112,23 +107,24 @@ if [[ -n $DSSTOPS ]]; then
   echo "Using user-provided input list of DS Stops $DSSTOPS"
 else
   DSSTOPS="CosmicDSStops.txt"
-  samListLocations ${SAMOPT} --defname="sim.mu2e.CosmicDSStops${S1NAME}${LOW}.${S1CONF}.art"  > ${DSSTOPS}
+  samweb list-definition-files sim.mu2e.CosmicDSStops${S1NAME}${LOW}.${S1CONF}.art  > ${DSSTOPS}
 fi
 
 if [ ! -f $DSSTOPS ]; then
   echo "Can't find DSStops"
   exit_abnormal
 fi
 
-cmd="generate_fcl --dsconf ${OUTCONF} --dsowner ${OWNER} --run-number ${RUNNUM} --description ${S2OUT} --embed ResampleS1.fcl
---events-per-job ${NEVTS} --njobs ${NJOBS} --auxinput=1:physics.filters.CosmicResampler.fileNames:${DSSTOPS}"
+cmd="mu2ejobdef --embed ResampleS1.fcl --setup /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/${S1CONF}/setup.sh --run-number=${RUNNUM} --events-per-job=${NEVTS} --desc ${S2OUT} --dsconf ${OUTCONF} --auxinput=1:physics.filters.CosmicResampler.fileNames:${DSSTOPS}"
```

Comment:
how will you test when developing (ie before there's a musing)?  Will you create a temporary musing?  Nominally things in /cvmfs are considered archival, do we want a temporary location for testing?

Reply:
We now test with the latest version of SimJob.
Local scripts have to be uploaded.
We can consider testing against nightly builds.